shared/AppInsightsCore/src/JavaScriptSDK/InstrumentHooks.ts

```diff
@@ -183,44 +227,7 @@ export function InstrumentFunc(target:any, funcName:string, callbacks: IInstrume
         if (owner) {
             let fn = owner[funcName]
             if (typeof fn === strShimFunction) {
```

Comment:
This was another issue, in that not all environments would return the "window.onerror" (and other events) as functions, they get returned as Object. Which lead to the hook never getting installed